JustSaying.Models/JustSaying.Models.csproj

```diff
@@ -1,9 +1,5 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.6;net451</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
Do you think the Models project/package should target netstandard2.0, given it only contains a single POCO? Seems like it could maybe even just target netstandard1.0?

Reply:
That's a good point. It seems like this package should be as portable as possible.

Reply:
Makes sense to target 1.0 if it doesn't depend on anything else.